src/traces/funnelarea/layout_attributes.js

```diff
@@ -0,0 +1,43 @@
+/**
+* Copyright 2012-2019, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var hiddenlabels = require('../pie/layout_attributes').hiddenlabels;
```

Comment:
It might be nice to add a line in the `hiddenlabels` description mentioning that it applies to `pie` and `funnelarea` labels simultaneously.

Reply:
Ah you did already. Thanks very much! Resolved.